ziti/cmd/create/config_templates/controller.yml

```diff
@@ -211,6 +223,8 @@ web:
         # These settings are per API. The example below is for the 'edge-api' and contains both optional values and
         # required values.
         options: { }
+      - binding: edge-oidc
```

Comment:
should this be if'ed the same way that lines 11-15 are?

Reply:
I am pretty sure you can have oidc without HA